modules/lib/src/main/resources/assets/js/app/inputtype/ui/text/dialog/LinkModalDialog.ts

```diff
@@ -800,11 +800,20 @@ export class LinkModalDialog
         });
 
         mediaUploader.onFileUploaded((event: UploadedEvent<Content>) => {
-            let item = event.getUploadItem();
-            let createdContent = item.getModel();
+            const mediaRadio = <RadioGroup>this.getFieldById('contentMediaRadio');
```

Comment:
I insist on us using static typing for all variables, even if the type can be inferred from looking on the var